packages/react-auth/src/AuthProvider/README.md

```diff
@@ -66,26 +66,26 @@ const App = () => {
   const navigate = useNavigate();
 
   if (loading) {
-    return (<div>Loading<div/>);
+    return <div>Loading</div>;
```

Comment:
The quotes and semicolons changes were done by Prettier. What I specifically fixed were these closing tags.